tests/cases/fourslash/codeFixClassImplementInterfaceAutoImports2.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference path='fourslash.ts' />
+
+// @importsNotUsedAsValues: error
+
+// @Filename: types1.ts
+////type A = {};
+////export default A;
+
+// @Filename: types2.ts
+////export type B = {};
+////export type C = {};
+////export type D<T> = {};
+
+// @Filename: interface.ts
+////import type A from './types1';
```

Comment:
`import type` is the only difference I could see between 1 and 2. Can you call it out in the filename?